### PR TITLE
Add array of accepted synonym packages for sheetify

### PR DESCRIPTION
**Pull request by bcomnes**
I'm looking to export sheetify from `hui` in order to enforce the same version of the tool across a codebase built on that component library.  This PR attempts to add an array similar to what is found in hyperhtml which would enable the ability to use package synonyms.

### CODE REVIEW

**Comment by bcomnes** [comment]
Ping @emilbayes @goto-bus-stop perhaps you would have thoughts on this.

File: transform.js [hunk 1]
```diff
@@ -73,7 +74,8 @@ function transform (filename, options) {
     var mname = null
     var ast
 
-    if (src.indexOf('sheetify') === -1) {
+    // Skip transforming any files that do not contain synonyms for sheetify
+    if (!SUPPORTED_VIEWS.some(name => src.indexOf(name) > -1)) {
```
Comment (emilbayes) [inline transform.js]: You can also use https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes to make intent just a little bit clearer

Comment (bcomnes) [reply transform.js]: Ah yeah, agreed.  Let me fix that.

File: transform.js [hunk 2]
```diff
@@ -110,7 +112,7 @@ function transform (filename, options) {
       if (node.type === 'CallExpression' &&
       node.callee && node.callee.name === 'require' &&
       node.arguments.length === 1 &&
-      node.arguments[0].value === 'sheetify') {
+      SUPPORTED_VIEWS.some(name => node.arguments[0].value === name)) {
```
Comment (emilbayes) [inline transform.js]: Same goes here :)

Comment (bcomnes) [reply transform.js]: Should be good now

**Comment by bcomnes** [comment]
@emilbayes ok updated.  wdyt?

**Comment by bcomnes** [comment]
Looks like includes would be a major version bump.  Fine with me.

File: transform.js [hunk 3]
```diff
@@ -10,6 +10,7 @@ const path = require('path')
 const fs = require('fs')
 
 const sheetify = require('./index')
+const SUPPORTED_VIEWS = require('./lib/supported-views')
```
Comment (goto-bus-stop) [inline transform.js]: maybe views is not the best name for this, i thought in nanohtml it's referring to view libraries

**Review by goto-bus-stop — APPROVED** [review]

File: transform.js (hunk 3 above)
Comment (bcomnes) [reply transform.js]: I agree, this isn't the best name for it.  Let me think of a better name.

Comment (bcomnes) [reply transform.js]: Ok I renamed this to `SUPPORTED_SYNONYMS`.

**Comment by bcomnes** [comment]
Tests pass up till node 6, so the major version bump should note that.

**Comment by bcomnes** [comment]
I'm going to release an 8.0.0-beta0 today and ensure this works properly in the wild.  After that, I will release the 8.0.0 with a CHANGELOG/release page.

**Comment by bcomnes** [comment]
Released in `8.0.0-beta.0`.  Install with `npm I sheetify@beta`.

**Pull request merged**